packages/core/src/__tests__/embrace.test.ts

```diff
@@ -250,41 +253,55 @@ describe("Logs Test", () => {
   test("logWarning", async () => {
     await logWarning("test message");
     expect(mockLogMessageWithSeverityAndProperties).toHaveBeenCalledWith(
-      `test message`,
+      "test message",
       WARNING,
       undefined,
-      mockSt,
+      MOCK_STACKTRACE,
     );
   });
 
   test("logError", async () => {
     await logError("test message");
     expect(mockLogMessageWithSeverityAndProperties).toHaveBeenCalledWith(
-      `test message`,
+      "test message",
       ERROR,
       undefined,
-      mockSt,
+      MOCK_STACKTRACE,
     );
   });
 });
 
-describe("Log handled Error Tests", () => {
-  const testError = new Error();
-  const testProps = {foo: "bar"};
-
-  test.each`
-    message           | properties   | out
-    ${"not an error"} | ${undefined} | ${{}}
-    ${testError}      | ${undefined} | ${{message: testError.message, stack: testError.stack}}
-    ${testError}      | ${testProps} | ${{message: testError.message, stack: testError.stack, properties: testProps}}
-  `("logHandledError", async ({message, out, properties}) => {
-    await logHandledError(message, properties);
-    // TODO uncomment the expect once the method is imeplemented
-    // if (message instanceof Error) {
-    //   expect(mockLogHandledError).toHaveBeenCalledWith(out.message, out.stack, out.properties);
-    // } else {
-    //   expect(mockLogHandledError).not.toHaveBeenCalled();
-    // }
+describe("Log Handled Errors", () => {
+  const testError = new Error("This is a test error");
+
+  it("error (instance of Error) with properties", async () => {
+    const testProps = {foo: "bar"};
+    await logHandledError(testError, testProps);
+
+    expect(mockLogHandledError).toHaveBeenCalledWith(
+      testError.message,
+      testError.stack,
+      testProps,
+    );
+  });
+
+  it("error (instance of Error) without properties", async () => {
+    await logHandledError(testError, undefined);
+
+    expect(mockLogHandledError).toHaveBeenCalledWith(
+      testError.message,
+      testError.stack,
+      {},
+    );
+  });
+
+  it("not an instance of error", async () => {
+    // even when ts complains about the type, we want to test this scenario
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    await logHandledError("not an error", undefined);
+
+    expect(mockLogHandledError).not.toHaveBeenCalled();
   });
 });
```

Comment:
I think could have one more test for the ios/android platform differences in how we call `logUnhandledJSException`